app/main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+import os
+import shutil
+
+
+class Errors(BaseException):
```

Comment:
Your own exceptions shouldn't be inherited from BaseException - they are too general. Don't overcomplicate this,  OsError will cover most cases